.github/workflows/release.yaml

```diff
@@ -28,10 +28,6 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: "1.18"
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-        with:
-          platforms: linux/amd64,linux/arm64/v8
```

Comment:
@bells17 
Why did you remove only this `setup-qemu-actions` and leave remaining in other workflows? AFAIU, `setup-qemu-action` does almost the same thing as `docker run tonistiigi/binfmt`, so either one of `setup-qemu-action` or `docker run tonistiigi/binfmt` is necessary. Why not ALL `setup-qemu-action` could be removed because of `make setup`?

Reply:
@daichimukai Sorry, I forgot to remove it in other actions.
I removed the setup-qemu-action in other actions.

Reply:
@daichimukai Sorry, I forgot to remove it in other actions.
I removed the setup-qemu-action in other actions.

Reply:
@bells17 
Thank you. The diff looks good to me. Please squash the commits.

Reply:
@daichimukai I squashed the commits. Please review this again.